### ISSUE TITLE
JAWS cannot read non-simple ListViewItems

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#8109]
### Describe the bug

ListViewItems that contain non-simple layouts (i.e. a grid with multiple text fields) are not read properly by JAWS. Specifically, only the first line of text is read, not the second. Surprisingly, this does not repro with NVDA or Narrator.

### Steps to reproduce the bug

1. Enable JAWS
2. In WinUI 2 gallery and WinUI 3 gallery, go to ListView page
3. Tab to "ListView with Selection Support" section
4. Use arrow keys to move between list view items

### Expected behavior

I expect all text in a list view item to be read. For example, the first entry would read "Kendall Collins. Adatum Corporation." Instead, only "Kendall Collins" is read.

### NuGet package version

None

### Additional context

- Originally found in https://github.com/microsoft/terminal/issues/14152
- Surprisingly, this issue does not repro on NVDA or Narrator. This means that it _could_ be a bug on JAWS as well, to be fair. But I've encountered cases in the past where JAWS is rightfully expecting something to be available and it's not.

**Comment by ranjeshj** [comment]
Following up with Narrator team on this.

**Comment by kmahone** [comment]
I was able to reproduce this in Narrator in WinUI3 Gallery.

For a ListView, the ListViewItem will use the data items ToString() method to create a UIA Name that will be read by the accessibility tool. WinUI Gallery is not including the full description of the item and so Narrator is reading an incomplete description. In Narrator it is still possible to directly navigate to the individual textblocks that are the children of the ListViewItem. However, using normal keyboard navigation will not hit those since they are not tab stops. 

I will make a fix for WinIU Gallery.

**Comment by kmahone** [comment]
Should be resolved with:
[Improve accessibility of ListView page - list view items UIA description includes more information. by kmahone · Pull Request #1435 · microsoft/WinUI-Gallery](https://github.com/microsoft/WinUI-Gallery/pull/1435)
